From 792e51318b8e7148ea53fb512ba434b77e0cff8a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 13 Oct 2020 20:22:10 +0300
Subject: [PATCH] [SW-12754] gaudi: security: rename pcie rrs func

The preboot_secure_pcie_lbw_rr() function can (and will) not only be
used for LBW, but also for HBW.
In order to avoid confusion and make the function name reflect better
its purpose and available functionality, rename to
preboot_secure_pcie_rr().

Change-Id: I0d251203ab7fa5c6d9d69fc776017734422e6f9a
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 15d89ef..47d9bb7 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -981,11 +981,10 @@
 	 */
 }
 
-static void preboot_secure_pcie_lbw_rr(int rr, u64 min64, u64 max64)
+static void preboot_secure_pcie_rr(int rr, u64 min64, u64 max64)
 {
 	u32 min = min64 & ~HL_CFG_BASE;
 	u32 max = max64 & ~HL_CFG_BASE;
-	/* We allocate 0 - 15 PCIe RRs for LBW */
 	u32 rr_base = rr * sizeof(u32), rr_dest;
 
 	/* Set WP RR */
@@ -1043,61 +1042,61 @@
 	min = mmSRAM_Y0_X0_BANK_BASE - 1;
 	max = mmSIF_RTR_0_BASE;
 	preboot_secure_lbw_rr(0, min, max);
-	preboot_secure_pcie_lbw_rr(0, min, max);
+	preboot_secure_pcie_rr(0, min, max);
 
 	/* CPU_TIMESTAMP: RR_1 */
 	min = mmCPU_TIMESTAMP_BASE - 1;
 	max = mmDMA_IF_W_S_BASE;
 	preboot_secure_lbw_rr(1, min, max);
-	preboot_secure_pcie_lbw_rr(1, min, max);
+	preboot_secure_pcie_rr(1, min, max);
 
 	/* HBM_CFG up to GIC_L: RR_2 */
 	min = mmHBM0_BASE - 1;
 	max = mmGIC_DISTRIBUTOR__5_GICD_SETSPI_NSR;
 	preboot_secure_lbw_rr(2, min, max);
-	preboot_secure_pcie_lbw_rr(2, min, max);
+	preboot_secure_pcie_rr(2, min, max);
 
 	/* GIC_H up to PCIE_WRAP: RR_3 */
 	min = mmGIC_DISTRIBUTOR__5_GICD_SETSPI_NSR + sizeof(u32) - 1;
 	max = mmPCIE_WRAP_BASE;
 	preboot_secure_lbw_rr(3, min, max);
-	preboot_secure_pcie_lbw_rr(3, min, max);
+	preboot_secure_pcie_rr(3, min, max);
 
 	/* PCIE_DBI up to MMU_UP: RR_4 */
 	min = mmPCIE_DBI_BASE - 1;
 	max = mmMMU_UP_BASE;
 	preboot_secure_lbw_rr(4, min, max);
-	preboot_secure_pcie_lbw_rr(4, min, max);
+	preboot_secure_pcie_rr(4, min, max);
 
 	/* PSOC_I2C_M0 up to PSOC_TIMESTAMP: RR_5 */
 	min = mmPSOC_I2C_M0_BASE - 1;
 	max = mmPSOC_TIMESTAMP_BASE;
 	preboot_secure_lbw_rr(5, min, max);
-	preboot_secure_pcie_lbw_rr(5, min, max);
+	preboot_secure_pcie_rr(5, min, max);
 
 	/* PSOC_GPIO0 up to NIC0_MAC_CH0: RR_6 */
 	min = mmPSOC_GPIO0_BASE - 1;
 	max = mmNIC0_MAC_CH0_BASE;
 	preboot_secure_lbw_rr(6, min, max);
-	preboot_secure_pcie_lbw_rr(6, min, max);
+	preboot_secure_pcie_rr(6, min, max);
 
 	/* NIC0_TS up to NIC1_MAC_CH0: RR_7 */
 	min = mmNIC0_TS_BASE - 1;
 	max = mmNIC1_MAC_CH0_BASE;
 	preboot_secure_lbw_rr(7, min, max);
-	preboot_secure_pcie_lbw_rr(7, min, max);
+	preboot_secure_pcie_rr(7, min, max);
 
 	/* NIC1_TS up to NIC2_MAC_CH0: RR_8 */
 	min = mmNIC1_TS_BASE - 1;
 	max = mmNIC2_MAC_CH0_BASE;
 	preboot_secure_lbw_rr(8, min, max);
-	preboot_secure_pcie_lbw_rr(8, min, max);
+	preboot_secure_pcie_rr(8, min, max);
 
 	/* NIC3_TS up to NIC4_MAC_CH0: RR_9 */
 	min = mmNIC3_TS_BASE - 1;
 	max = mmNIC4_MAC_CH0_BASE;
 	preboot_secure_lbw_rr(9, min, max);
-	preboot_secure_pcie_lbw_rr(9, min, max);
+	preboot_secure_pcie_rr(9, min, max);
 }
 
 static void preboot_secure_hbw_rr(int rr, u64 base, u64 mask)
